sources/core-sw/src/compression/opt/placeholder.h

```diff
@@ -4,7 +4,12 @@
  * SPDX-License-Identifier: MIT
  ******************************************************************************/
 
+#ifndef PLACEHOLDER_H
+#define PLACEHOLDER_H
```

Comment:
I understand that QPL doesn't follow it everywhere, but in order to improve, could we follow the convention `QPL_<dir_name_1>_<dir_name_2>_..._<dirN>_<header_file_name>_H/HPP`?

Reply:
no problem. will add a config policy around this and regenerate the branch.